fern/main.css

```
@@ -0,0 +1,1351 @@
+.copyright {
+    color: var(--grayscale-a11);
+    font-size: 0.875rem;
+    line-height: 1.65;
+    margin: 0;
+}
+@media (max-width: 768px) {
+    .bd-footer { padding: 1.5rem 0; }
+    .bd-footer__inner { padding: 0 1.5rem; }
+    .footer-items__start { gap: 1rem; }
+    .footer-links { flex-direction: row; gap: 0.5rem 0.75rem; }
+    .footer-links a { white-space: normal; word-break: break-word; }
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)